### DISCUSSION TITLE
An AttributeError on Ordinary Percolation

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2354] [Q&A] [answered]
Hi, professor. Recently, I have been studying the "Ordinary Percolation" and the website address is https://github.com/PMEAL/OpenPNM/blob/dev/examples/simulations/percolation/A_ordinary_percolation.ipynb . An AttributeError , 'SettingsDict' object has no attribute '_update', occurred when I ran the codes. I don't know what‘s wrong.

**Comment by ma-sadeghi** [comment, marked as answer]
We're transitioning to v3. Until then, most examples won't work with the version you find on PyPI or conda. You can either wait until then, or just use the development branch. [Here](https://openpnm.org/user_guide/installation.html#installing-the-dev-version), you can find instructions on how to install the dev version.
